exercises/nucleotide-count/src/example.c

```diff
@@ -0,0 +1,44 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include "nucleotide_count.h"
+
+char *count(const char *dna_strand)
+{
+   bool invalidChar = false;
+   uint16_t index;
```

Comment:
I realize that it's way overkill, but why not `size_t` here? This is guaranteed to be capable of handling any string whereas `uint16_t` seems a bit arbitrary.